code/controllers/subsystem/persistence.dm

```diff
@@ -104,6 +106,15 @@ SUBSYSTEM_DEF(persistence)
 		return
 	saved_modes = json["data"]
 
+/datum/controller/subsystem/persistence/proc/LoadRecentMaps()
+	var/json_file = file("data/RecentMaps.json")
```

Comment:
Define for this, also do we not have a variable for `data/`?

Reply:
We don't

Reply:
Heck, copy pasted LoadRecentModes but didnt scrutinize it. Ill fix tomorrow

Reply:
Would have fixed the rest while I was at it, but since you're touching this in #48448 I decided not to.